docs/topic_guides/evaluation/clusters/graph_metrics.md

```diff
@@ -0,0 +1,91 @@
+
+Bear in mind, that the degree of a single node in a cluster isn't necessarily representative of the overall connectedness of a cluster. This is where [cluster centralisation]() can help.
+
+## 🔗 Edge metrics
+
+Edge metrics quantify the properties of the edges within a cluster. 
+
+### Example: 'is bridge'
+
+An edge is classified as a 'bridge' if its removal splits a cluster into two smaller clusters.
+
+[insert picture]
+
+Bridges can be signalers of false positives in linked data, especially when joining two highly connected sub-clusters. Examining bridges can shed light on potential errors in the linking process leading to the formation of false positive links.
+
+## :fontawesome-solid-circle-nodes: Cluster metrics
+
+Cluster metrics refer to the characteristics of a cluster as a whole, rather than the individual nodes and edges it contains.
+
+### Example: cluster size
+
+Cluster size refers to the number of nodes within a cluster.
+
+When thinking about cluster size, it is important to consider the size of the biggest clusters produced and ask yourself, does this seem reasonable for the dataset being linked? For example, does it make sense that one person is appearing hundreds of times in the linked data resulting in a cluster of over 100 nodes? If the answer is no, then false positives links are probably being formed. This could be, for example, due to having blocking rules which are too loose.
+
+If you don't have an intuition of what seems reasonable, then it is worth inspecting a sample of the largest clusters in Splink's [Cluster Studio Dashboard]() to validate (or invalidate) links. From there you can develop an understanding of what maximum cluster size to expect.
+
+There also might be a lower bound on cluster size. For example, when linking two datasets in which you know people appear least once in each, the minimum expected size of cluster will be 2. Clusters smaller than the minimum size indicate links have been missed. This could be due to blocking rules not letting through all record comparisons of true matches.
+
+### Example: cluster density
+
+The density of a cluster is given by the number of edges it contains divided by the maximum possible number of edges. Density ranges from 0 to 1. A density of 1 means that all nodes are connected to all other nodes in a cluster.
+
+[picture: edges vs max possible edges]
+
+When evaluating clusters, a high density (closer to 1) is generally considered good as it means there are many edges in support of the records in a cluster being linked.
+
+A low density could indicate links being missed. This could happen, for example, if blocking rules are too tight or the clustering threshold is too high.
```

Comment:
Maybe low density could also be if you have false positives. - a few dense true clusters connected by a few false positives?

Reply:
I think I might add in something on this under cluster centralisation - high centralisation + low density could indicate a few FPs between highly connected clusters

Reply:
Had another think about this. High cluster centralisation + low density could also indicate a star graph without any 'multi-bridges'. I think it makes sense to leave this for the next topic guide iteration when we have a better understanding of how to use the metrics in combination.